src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.SslProvider.cs

```diff
@@ -95,7 +95,12 @@ internal static void SetSslOptions(EasyRequest easy, ClientCertificateOption cli
             private static void SetSslVersion(EasyRequest easy, IntPtr sslCtx = default(IntPtr))
             {
                 // Get the requested protocols.
-                System.Security.Authentication.SslProtocols protocols = easy._handler.ActualSslProtocols;
+                SslProtocols protocols = easy._handler.SslProtocols;
```

Comment:
> If it's internal shouldn't it be capitalized

Our guidelines are that all non-public instance fields are prefix with an underscore, regardless of private or internal.

>  and, even better, converted into a property to preserve encapsulation and the open-close principle? (easy.Handler instead of easy._handler)

My care level on that, for an internal implementation detail where this is essentially just a bag of state shared between different pieces of the same type, is basically zero.  If you'd like to submit a PR to make the change, I'd be fine with it.  Regardless, that naming exists well earlier than this change.